dotcom-rendering/src/components/FollowButtons.tsx

```diff
@@ -0,0 +1,127 @@
+import { css } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { space, textSans } from '@guardian/source-foundations';
+import {
+	SvgCheckmark,
+	SvgNotificationsOff,
+	SvgNotificationsOn,
```

Comment:
There’s an upcoming change in Source that will bring the rounded icons directly: https://github.com/guardian/csnx/pull/939#issuecomment-1822543330

Reply:
we can replace these when we do the wider byline ui changes 👍